calico/felix/endpoint.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, config, ip_type,
 
         # All endpoint dicts that we know about.
         self.endpoints_by_id = {}
+        # MD4: Why do we need this?  There's already an objects_by_id (and it's
```

Comment:
endpoints_by_id is full of endpoint dicts.  objects_by_id contains the active LocalEndpoint objects that have been ref-counted into existence.